trnsysGUI/GUI.py

```diff
@@ -739,4 +740,9 @@ def main():
 
 
 if __name__ == "__main__":
+    if pkg_resources.get_distribution("pytrnsys-gui").parsed_version.local.endswith('dev'):
```

Comment:
Ah, so this works now? Great! What did we miss last time we talked about it?

Let's be generous and give this bit of code its own function with a meaningful name and maybe introduce some local variables on top? ;)

Reply:
...and then maybe call this function first thing in `main`?

Reply:
Sure.  Last time we only had pkg_resources.get_distribution("pytrnsys-gui").endswith('dev') but it did not work.

Reply:
Aha, I see.